go.mod

```diff
@@ -162,9 +162,9 @@ replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.3
 replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.3-patched
 
 // TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
-replace github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.44.6-cronos-revert
+replace github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.44.6-cronos-revert-2
```

Comment:
is there any differences?

Reply:
https://github.com/crypto-org-chain/cosmos-sdk/compare/v0.44.6-cronos-revert...v0.44.6-cronos-revert-2
yes, added a public method to the baseapp

Reply:
looks good